### ISSUE TITLE
createClass is deprecated in React 15.5

### ISSUE THREAD

**Issue by suchipi** [JedWatson/react-hammerjs#66]
As of React 15.5, createClass is deprecated, so using this library prints the following in the console:
```
Warning: Hammer: React.createClass is deprecated and will be removed in version 16. Use plain JavaScript classes instead. If you're not yet ready to migrate, create-react-class is available on npm as a drop-in replacement.
```
Hammer should either switch to using `create-react-class` or use plain JavaScript classes.

**Comment by afc163** [comment]
ping~

**Comment by JedWatson** [comment]
Hopefully this will be addressed by my suggested changes to #67

**Comment by ebhoren** [comment]
+1

**Comment by raulrene** [comment]
+ Number.MAX_SAFE_INTEGER

**Comment by renatorib** [comment]
+1

**Comment by pedronauck** [comment]
+1

**Comment by myrubapa** [comment]
+1

**Comment by renatorib** [comment]
+2

**Comment by jrsinclair** [comment]
This just became slightly more pressing, now that React 16 has been released.

**Comment by JedWatson** [comment]
Fixed as of `1.0.1`
